governance/xc_admin/packages/xc_admin_frontend/hooks/useMultisig.ts

```diff
@@ -73,21 +75,31 @@ export const useMultisig = (): MultisigHookData => {
         })
       )
     } else {
-      setSquads(undefined)
+      setSquads(
+        new SquadsMesh({
+          connection,
+          wallet: new NodeWallet(new Keypair()),
+        })
+      )
```

Comment:
I'm a bit confused why we even have `readOnlySquads` if `squads` falls back to the same value, can't we just use `squads` everywhere?

Reply:
It's because I don't want to reload all the data when a user connects the wallet.